Khan/perseus
# show square options instead of circles when the radio widget is a multiselect

## Summary:
Makes the radio widget look more like a checkbox when the user is being prompted to select multiple options.

Issue: LP-11581

## Test plan:
- go to a page that has an exercise with a radio widget where a user can select multiple options
- confirm that the radio widget choices look more square-ish
- go to an exercise where you can only select one option
- confirm that the choices look circle-y

Library
<img width="381" alt="Screen Shot 2022-09-29 at 1 46 35 PM" src="https://user-images.githubusercontent.com/16308368/193117043-6de5126e-3519-4884-92ee-ecc3fc9bd31f.png">
<img width="383" alt="Screen Shot 2022-09-29 at 1 45 57 PM" src="https://user-images.githubusercontent.com/16308368/193117046-92ba4ae1-6903-415d-8f9f-cbfc4f072865.png">

SAT
<img width="365" alt="Screen Shot 2022-09-29 at 1 46 46 PM" src="https://user-images.githubusercontent.com/16308368/193117041-2cd8ec88-7465-459a-b13f-f9e20d65edc0.png">
<img width="362" alt="Screen Shot 2022-09-29 at 1 46 16 PM" src="https://user-images.githubusercontent.com/16308368/193117044-8f419426-636d-4949-a4fc-76ebc685b24c.png">
